webpack.config.js

```diff
@@ -0,0 +1,41 @@
+const webpack = require('webpack');
+
+module.exports = {
+    entry: './src/index.js',
+    module: {
+        rules: [{
+            test: /\.(js|jsx)$/,
+            exclude: /node_modules/,
+            use: ['babel-loader']
+        },
+        {
+            test: /\.(css|scss)$/,
+            use: [
+                'style-loader',
+                'css-loader',
+                'sass-loader'
+          ]  
+        },
+        {
+            test: /\.(jpg|jpeg|mp3|gif|png|svg)$/,
+            loaders: ['file-loader']
+        }]
+    },
+      resolve: {
+        extensions: ['*', '.js', '.jsx']
+      },
+    output: {
+      path: __dirname + '/public',
+      publicPath: '/',
+      filename: 'bundle.js'
+    },
+    mode: process.env.NODE_ENV || 'development',
+    plugins: [
+        new webpack.HotModuleReplacementPlugin()
+    ],
+    devServer: {
+      contentBase: './public',
+      hot: true
+    }
+  };
+  
```

Comment:
Great job Bukky. Please include the EOL on this file also.

Reply:
Done